faq.md

```diff
@@ -92,13 +92,13 @@ This is a lightweight option of having multiple clusters for different users, si
 There is regex subscription coming up in Pulsar 2.0. See [PIP-13](https://github.com/apache/pulsar/wiki/PIP-13:-Subscribe-to-topics-represented-by-regular-expressions).
 
 ### Does Pulsar have, or plan to have, a concept of log compaction where only the latest message with the same key will be kept ?
-Yes, see [PIP-9](https://github.com/ivankelly/pulsar-wiki/pull/1/files) for more details.
+Yes, see [PIP-6](https://github.com/apache/pulsar/wiki/PIP-6:-Guaranteed-Message-Deduplication) for more details.
```

Comment:
I don't think this is the right PIP. It should be PIP-14.

Reply:
@sijie Please review again.